tests/authentication/test_authentication_blank_field.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+from allure import description, step, title, feature
+from hamcrest import assert_that, is_, contains_string
+import re
+
+from framework.endpoints.authenticate_api import AuthenticateAPI
+from framework.tools.generators import generate_user_data
+
+timestamp_pattern = r"\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}"
+
+
+@feature("Authentication of user")
+class TestAuthentication:
+    @title("Test authentication, negative scenario")
+    @description(
+        "GIVEN user is registered"
+        "WHEN fields password, email are blank "
+        "THEN status HTTP CODE = 400"
+    )
+    @pytest.mark.parametrize("email, password, expected_status_code, expected_message_part", [
+        (" ", " ", 400, "Email is the mandatory attribute"),
+        (None, " ", 400, "Password is the mandatory attribute"),
+        (" ", None, 400, "Email is the mandatory attribute")
+    ], ids=["password_and_email_blank", "password_blank", "email_blank"])
```

Comment:
I suggest using https://pytest.org/en/latest/example/parametrize.html#different-options-for-test-ids
```
@pytest.mark.parametrize(
    "email, password, expected_status_code, expected_message_part",
    [
        pytest.param(" ", " ", 400, "Email is the mandatory attribute", id="password_and_email_blank"),
...
    ],
)
```

Reply:
Fixed